flowmachine/tests/conftest.py

```diff
@@ -98,8 +98,7 @@ def pytest_itemcollected(item):
     and also adds info about any parameters (if the test is parametrized).
     """
     if item.obj.__doc__:
-        item._nodeid = "* " + " ".join(item.obj.__doc__.split())
-        item._nodeid += get_string_with_test_parameter_values(item)
```

Comment:
Did you intentionally drop the info about parameter values? I think I found this useful in the past to debug parametrised tests where it only fails for certain values.

But if you do want to drop it then maybe we should remove the function definition for`get_string_with_test_parameter_values()` because it's now not used anywhere any more?

Also, the exact same code snippet also lives in `flowetl/tests/conftest.py`, so would be worth porting your improvements to that conftest as well.

Reply:
Pytest actually includes the params in the report for failed tests anyway :)